packages/turbo/src/VisitableView.tsx

```diff
@@ -151,8 +162,16 @@ const VisitableView = React.forwardRef<RefObject, React.PropsWithRef<Props>>(
       onConfirm
     );
 
-    const { loadingComponent, handleShowLoading, handleHideLoading } =
-      useRenderLoading(renderLoading);
+    const {
+      loadingComponent,
+      handleShowLoading: handleShowLoadingCallback,
+      handleHideLoading,
+    } = useRenderLoading(renderLoading);
+
+    const handleShowLoading = useCallback(() => {
+      hideErrorComponent();
+      handleShowLoadingCallback();
+    }, [handleShowLoadingCallback, hideErrorComponent]);
```

Comment:
This part of code fixes an issue shown on a video:

https://github.com/software-mansion-labs/react-native-turbo-demo/assets/44744462/5064ce73-fad2-479b-934e-800561c951f0